modules/server/Server.cpp

```diff
@@ -4,34 +4,137 @@
 
 #include <fstream>
 
-Connection *Server::simulate_connect(int socket) {
-	Connection *connection = new Connection(socket, sockaddr());
-	connections[socket] = connection;
-	return connection;
+#define MAX_BUFFER_LENGTH 4096
```

Comment:
```suggestion
#define MAX_BUFFER_LENGTH 512
```

"these messages SHALL NOT exceed 512 characters in length" RFC 2812